docs/src/docs/hooks/use-mouse.mdx

```diff
@@ -0,0 +1,53 @@
+---
+group: 'mantine-hooks'
+package: '@mantine/hooks'
+title: 'use-mouse'
+category: 'dom'
+order: 1
+slug: /hooks/use-mouse/
+description: 'Get mouse coordinates relatively to the viewport'
+import: "import { useMouse } from '@mantine/hooks';"
+docs: 'hooks/use-mouse.mdx'
+source: 'mantine-hooks/src/use-mouse/use-mouse.ts'
+---
+
+import { useMouseUsage, useMouseRef } from '@docs/demos';
+
+## Usage
+
+use-mouse returns moure x, y coordinates relatively to viewport screen:
+
+<Demo data={useMouseUsage} />
+
+## API
+
+Hook returns object with ref and x, y mouse coordinates:
+
+```tsx
+const {
+  ref, // -> pass ref to target element - optional in use
```

Comment:
"pass ref to target element - optional in use" should be "pass ref to target element, if not used document element will be used as target element"